### PR TITLE
chore: update markets nav item 2 columns

### PR DESCRIPTION
**Pull request by mohsen-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/components/organisms/navigation/nav-sections-container/section-column.tsx [hunk 1]
```diff
@@ -35,18 +36,22 @@ const NavSectionColumn = ({ item }: NavSectionColumnProps) => {
...
+                    <Typography.Paragraph
+                        className={styles.section_items_title}
+                        textcolor="secondary"
+                        align="left"
+                        font_family="UBUNTU"
+                        mb="2x"
+                        visible={!item.data.title ? 'larger-than-tablet' : undefined}
+                    >
+                        {item.data.title ? <Localize translate_text={item.data.title} /> : ''}
+                    </Typography.Paragraph>
```
Comment (NikitK-deriv) [inline src/features/components/organisms/navigation/nav-sections-container/section-column.tsx:53]: Can we use `&&`?
```suggestion
                        visible={!item.data.title && 'larger-than-tablet'}
                    >
                        {item.data.title && <Localize translate_text={item.data.title} />}
                    </Typography.Paragraph>
```

File: src/features/components/templates/navigation/template/desktop-menu/styles.scss [hunk 2]
```diff
@@ -51,7 +51,7 @@
         justify-content: end;
         perspective: 2000px;
         position: absolute;
-        inset-inline-end: -2rem;
+        inset-inline-end: -10rem;
```
Comment (NikitK-deriv) [inline src/features/components/templates/navigation/template/desktop-menu/styles.scss:54]: Isn't the same with `inset-inline-start: 10rem` ?

File: src/features/components/organisms/navigation/nav-sections-container/section-column.tsx (hunk 1 above)
Comment (mohsen-deriv) [reply src/features/components/organisms/navigation/nav-sections-container/section-column.tsx:53]: hey man, no we can't we have to provide the empty string. if you check the trade dropdown in ROW, you'll see the empty string.

File: src/features/pages/home/live-pricing/market-tabs/market-tab.tsx [hunk 3]
```diff
@@ -26,28 +29,26 @@ const MarketTab = () => {
...
+                            onMarketButtonClick(market_item.market_name)
+                            setLinkToMarketPage(market_item.url)
+                        }}
+                    >
+                        <TabMenu
+                            tab_names={[market_item.button_text]}
+                            current_tab={tab_name}
+                            setCurrentTab={setTabName}
+                            icon={
+                                market_item.market_name === selected_market
```
Comment (habib-deriv) [inline src/features/pages/home/live-pricing/market-tabs/market-tab.tsx:46]: Can we get this logic out of here and use as a variable?

File: src/features/pages/home/live-pricing/market-tabs/utils.tsx
Comment (habib-deriv) [inline src/features/pages/home/live-pricing/market-tabs/utils.tsx:1]: Hi @mohsen-deriv ,
Can we have a different file for content instead of utils?

File: src/features/pages/home/live-pricing/table-component/live-market-table.tsx
Comment (habib-deriv) [inline src/features/pages/home/live-pricing/table-component/live-market-table.tsx:1]: Is this file change related for the market navs change, maybe can we do thi in different PR?

File: src/features/components/templates/tabs/menu/index.tsx [hunk 4]
```diff
@@ -22,23 +22,21 @@ const TabMenu = ({ class_name, tab_names, current_tab, icon, setCurrentTab }: Ta
...
+                    <Tab.MenuItem
+                        key={tab_name}
+                        selected={tab_name == current_tab}
+                        onClick={() => {
+                            setCurrentTab(tab_name)
+                        }}
+                    >
+                        {icon && (
+                            <FlexBox.Box justify="center" padding_block="6x">
+                                {icon && <Image src={icon} width="24px" height="24px" />}
```
Comment (habib-deriv) [inline src/features/components/templates/tabs/menu/index.tsx:34]: Let's add some alt tag
